.travis.yml

```diff
@@ -2,6 +2,10 @@ language: python
 sudo: false
 python:
     - "2.7"
+env:
+  global:
+    - CF_DEPLOY_USER=fec_deployer
```

Comment:
I see this was already here below and I venture to guess this is a moot point, but don't we have this also set as a secure variable in our other Travis configs?